medmodels/medrecord/medrecord.py

```diff
@@ -1301,3 +1302,176 @@ def __getitem__(
+                    pl.all(),
+                ]
+            )
+
+            node_groups.append(node_info)
+
+        node_table = pl.concat(node_groups)
+
+        if node_table.is_empty():
+            node_table = pl.DataFrame({col: "-" for col in node_table.columns})
+
+        return node_table
+
+    def _describe_group_edges(self) -> pl.DataFrame:
+        """Creates a summary of edges connecting group nodes and the edge attributes.
+
+        Returns:
+            pl.DataFrame: DataFrame with an overview of edges connecting group nodes.
+        """
+
+        edge_groups = [
+            pl.DataFrame(
+                schema={
+                    "Edges": pl.String,
+                    "Count": pl.Int32,
+                    "Attribute": pl.String,
+                    "Info": pl.String,
+                }
+            )
+        ]
+
+        groups = sorted(set(self.groups))
+
+        for source_group in groups:
+            # edges in groups
+            edges = self.group(source_group)["edges"]
+
+            if edges:
+                schema = (
+                    self.schema.group(source_group).edges
+                    if source_group in self.schema.groups
+                    else None
+                )
+
+                edge_info = extract_attribute_summary(self.edge[edges], schema=schema)
+
+                edge_info = edge_info.select(
+                    [
+                        pl.lit(source_group).alias("Edges"),
+                        pl.lit(len(edges)).alias("Count"),
+                        pl.all(),
+                    ]
+                )
+
+                edge_groups.append(edge_info)
+
+            # edges connecting different groups
+            for target_group in groups:
+                edges = self.select_edges(
+                    edge().connected_source_with(node().in_group(source_group))
+                    & edge().connected_target_with(node().in_group(target_group))
+                )
+
+                if not edges:
+                    continue
+
+                edge_info = extract_attribute_summary(self.edge[edges])
+
+                edge_info = edge_info.select(
+                    [
+                        pl.lit(f"{source_group} -> {target_group}").alias("Edges"),
+                        pl.lit(len(edges)).alias("Count"),
+                        pl.all(),
+                    ]
+                )
+
+                edge_groups.append(edge_info)
+
+        edge_table = pl.concat(edge_groups)
+
+        if edge_table.is_empty():
+            edge_table = pl.DataFrame({col: "-" for col in edge_table.columns})
+
+        return edge_table
+
+    def __repr__(self) -> str:
+        representation = prettify_table(self._describe_group_nodes())
+        representation.append("")
+        representation.extend(prettify_table(self._describe_group_edges()))
+
+        return "\n".join(representation)
+
+    def print_attribute_table_nodes(self) -> None:
```

Comment:
I dont quite like the name of the function. Maybe?
```suggestion
    def print_table_nodes(self) -> None:
```

Reply:
that's what I had before and Jakob didn't like it :D